app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -194,21 +184,30 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
             ((DocumentActivity) activity).setDocumentTitle(_document.getTitle());
         }
 
+        _hlEditor.setSaveInstanceState(false); // We will reload from disk
+        _document.resetChangeTracking(); // force next reload
+        loadDocument();
```

Comment:
These cleanups / reorganization of the Document fragment help reduce how often the blank space issue happens. But it isn't totally solved yet.